specs/waku/v2/waku-filter.md

```diff
@@ -74,19 +67,17 @@ message FilterRPC {
 }
 ```
 
-##### FilterRPC
+#### FilterRPC
 
 A node MUST send all Filter messages (`FilterRequest`, `MessagePush`) wrapped inside a
 `FilterRPC` this allows the node handler to determine how to handle a message as the Waku
 Filter protocol is not a request response based protocol but instead a push based system.
 
-The `request_id` MUST be a uniquely generated string.
-
-##### FilterRequest
-
-TODO Specify mechanism for telling it won't honor (normal-no service-spam case)
+The `request_id` MUST be a uniquely generated string. When a `MessagePush` is sent
+the `request_id` MUST match the `request_id` of the `FilterRequest` whose filters
```

Comment:
What stops a node from sending messages with invalid `contentTopics` but correct `request_id`? Don't we still have to check the contentTopics?

Reply:
@kdeme I guess yeah

Reply:
Is there still a use for this if we have to check all topics anyhow?

Reply:
@kdeme wdym? the requestId? I think so yes. It seems like its pretty standard across our protocols now so even if just for that we should probably keep it.

Reply:
I didn't mean for having a requestId. I mean specifically for returning that requestId in the MessagePush